### ISSUE TITLE
symlinks value seems corrupted.

### ISSUE THREAD

**Issue by ruzia** [ajgon/opsworks_ruby#232]
After 1.17.0, symlinks value seems corrupted.

### Expected Behavior
Default value applied by https://github.com/ajgon/opsworks_ruby/blob/master/attributes/default.rb#L30-L57

### Actual Behavior
```{"dot_env"=>".env"}```

I think it cause by below codes.
https://github.com/ajgon/opsworks_ruby/blob/master/libraries/drivers_appserver_base.rb#L108
https://github.com/ajgon/opsworks_ruby/blob/master/recipes/deploy.rb#L50

First one sets to ```globals(:symlinks, application['short name'])```, so second one is always return ```{"dotenv"=>".env"}```.

**Comment by ajgon** [comment]
There was a [breaking change introduced at 1.17](https://github.com/ajgon/opsworks_ruby/blob/master/CHANGELOG.md#breaking-changes) which requires you, to provide ALL mappings if you are going to overwrite symlink. They are not appended anymore.

**Comment by ruzia** [comment]
Thanks for reply!

>to provide ALL mappings if you are going to overwrite symlink.

I don't overwrite any symlinks, just want to use default settings.
But symlinks only ```{"dot_env"=>".env"}``` because drivers_appserver_base.rb#L108 code.

**Comment by ajgon** [comment]
Weird, it shouldn't happen this way - can you provide me your custom JSON (with all sensitive data cleaned) ? I'll try to reproduce it.

**Comment by ruzia** [comment]
Sure.

Stack custom json
```
{
    "ruby-version": "2.6",
    "use-nodejs": true,
}
```

Layer custom json
```
{
    "deploy": {
        "rails": {
            "global": {
                "environment": "staging"
            },
            "source": {
                "remove_scm_files": false
            },
            "database": {
                "adapter": "postgresql",
                "username": "xxx",
                "password": "xxx",
                "host": "xxxx.ap-northeast-1.rds.amazonaws.com",
                "database": "xxx"
            },
            "framework": {
                "envs_in_console": true
            },
            "appserver": {
                "adapter": "passenger",
                "dot_env": true
            },
            "webserver": {
                "adapter": "apache2",
                "keepalive_timeout": 300,
                "limit_request_body": 100000000
            },
            "worker": {
                "adapter": "delayed_job",
                "process_count": 1
            }
        }
    }
}
```

Please tell me if I make a mistake.

**Comment by ajgon** [comment]
Ah, I see the problem now - you are absolutely correct, `dot_env` parameter adds new row to the symlinks, and with new behavior (override instead of append), it's the only one line which gets parsed. I'm quite busy right now, but I'll try to fix it to the end of the week.

In a meantime, can you use this workaround?

```
{
    "deploy": {
        "rails": {
            "global": {
                "environment": "staging",
                "symlinks": {
                    "system": "public/system",
                    "assets": "public/assets",
                    "cache": "tmp/cache",
                    "pids": "tmp/pids",
                    "log': "log"
                }
            },
            "source": {
                "remove_scm_files": false
            },
            "database": {
                "adapter": "postgresql",
                "username": "xxx",
                "password": "xxx",
                "host": "xxxx.ap-northeast-1.rds.amazonaws.com",
                "database": "xxx"
            },
            "framework": {
                "envs_in_console": true
            },
            "appserver": {
                "adapter": "passenger",
                "dot_env": true
            },
            "webserver": {
                "adapter": "apache2",
                "keepalive_timeout": 300,
                "limit_request_body": 100000000
            },
            "worker": {
                "adapter": "delayed_job",
                "process_count": 1
            }
        }
    }
}
```

This would basically "repopulate" the symlinks map, which would allow `dot_env: true` safely append to it.

**Comment by ruzia** [comment]
It's works.
I'd like to use webpacker, so I added node_modules and packs too.

**Comment by ajgon** [comment]
I released v1.18.0 of the cookbook, which (among the others) should include fix for this issue.

Thank you for catching this!
